Source: Peter Volkov <pva@gentoo.org>
Upstream: Notified
Reason: parallel make failed.

From automake info page:

   When writing `install-exec-hook' or `install-data-hook', please bear in mind
   that the exec/data distinction is based on the installation directory, not
   on the primary used (*note Install::).  So a `foo_SCRIPTS' will be installed
   by `install-data', and a `barexec_SCRIPTS' will be installed by
   `install-exec'.  You should define your hooks consequently.

   Variables using the standard directory prefixes `bin', `sbin', `libexec',
   `sysconf', `localstate', `lib', or `pkglib' are installed by `install-exec'.

thus change from install-data-hook to install-exec-hook.

   Automake also supports two install hooks, `install-exec-hook' and
   `install-data-hook'.  These hooks are run after all other install rules of
   the appropriate type, exec or data, have completed.  So, for instance, it is
   possible to perform post-installation modifications using an install hook.

thus no need for dependencies.

--- conf/Makefile.am	2008-09-02 14:26:04 +0000
+++ conf/Makefile.am	2008-09-02 17:18:02 +0000
@@ -4,7 +4,7 @@
 
 DEFAULT_CONFIG_FILE = $(sysconfdir)/GeoIP.conf
 
-install-data-hook: install-sysconfDATA
+install-exec-hook:
 	@if test -f $(DESTDIR)$(DEFAULT_CONFIG_FILE) ; then \
 	        echo "$@ will not overwrite existing $(DESTDIR)$(DEFAULT_CONFIG_FILE)" ; \
 	else \

--- data/Makefile.am	2008-09-02 14:26:04 +0000
+++ data/Makefile.am	2008-09-02 17:19:00 +0000
@@ -4,7 +4,7 @@
 
 DEFAULT_DB_FILE = $(pkgdatadir)/GeoIP.dat
 
-install-data-hook: install-pkgdataDATA
+install-data-hook:
 	@if test -f $(DESTDIR)$(DEFAULT_DB_FILE) ; then \
 	        echo "$@ will not overwrite existing $(DESTDIR)$(DEFAULT_DB_FILE)" ; \
 	else \

--- man/Makefile.am	2008-09-02 14:26:04 +0000
+++ man/Makefile.am	2008-09-02 17:19:06 +0000
@@ -5,6 +5,6 @@
 UPDATE_MAN = $(mandir)/man1/geoipupdate.1
 LOOKUP_MAN = $(mandir)/man1/geoiplookup.1
 
-install-data-hook: install-man
+install-data-hook:
 	cat geoipupdate.1 | sed s,DATADIR,$(pkgdatadir), | sed s,CONF_DIR,$(sysconfdir), > $(DESTDIR)$(UPDATE_MAN)
 	cat geoiplookup.1 | sed s,DATADIR,$(pkgdatadir), > $(DESTDIR)$(LOOKUP_MAN)

